### PR TITLE
live indicator of lives before game starts + css

### PR DESCRIPTION
**Pull request by DevUsman-18**

### CODE REVIEW

**Comment by Reductito** [comment]
on small screen sizes the pre game attempt number is centred but the input values aren't, could text align them at relevant media queries

File: index.html [hunk 1]
```diff
@@ -30,6 +30,10 @@ <h2>Instructions:</h2>
                         <label for="numLynx">How many Lynx?</label>
                         <input type="number" class="inputBox" id="numLynx" name="numLynx" min="1" max="50" required>
                     </div>
+                    <div class="userInput">
+                        <label for="preLives">You will have this many lives:</label>
```
Comment (dmayarace) [inline index.html]: Maybe a shorter way of saying this. Possibly 'Number of lives: '

Comment (dmayarace) [reply index.html]: Revisit the instructions to introduce this idea of lives lost. Line 20 'Click on the squares to find all the lynx in the forest... but beware of the snakes! You will lose a life each time you get a bite'

File: index.js [hunk 2]
```diff
@@ -114,6 +113,9 @@ function newGame() {
     //hides modal
     modalDiv.style.display = "none"
     modalBlur.style.filter = "none"
+
+    //reset preLives counter
+    document.getElementById("preLives").textContent = "-"
```
Comment (dmayarace) [inline index.js]: Maybe have this anytime the input is blank

File: index.js [hunk 3]
```diff
@@ -127,6 +129,14 @@ function generateLives(numLynx) {
     return lives
 }
 
+document.getElementById('numLynx').addEventListener("input", () => {
+    let lynxInputValue = parseInt(document.getElementById("numLynx").value)
+    preLives = generateLives(lynxInputValue)
+    if(preLives > 0){
+        document.getElementById("preLives").textContent = preLives
```
Comment (dmayarace) [inline index.js:146]: If prelives = empty string '- '

File: index.js [hunk 4]
```diff
@@ -127,6 +129,14 @@ function generateLives(numLynx) {
     return lives
 }
 
+document.getElementById('numLynx').addEventListener("input", () => {
+    let lynxInputValue = parseInt(document.getElementById("numLynx").value)
+    preLives = generateLives(lynxInputValue)
+    if(preLives > 0){
```
Comment (dmayarace) [inline index.js]: space between if and (

**Review by dmayarace — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (DevUsman-18) [reply index.js]: godayum

**Comment by lilyboughton** [comment]
We're happy :) nice stuff

**Pull request merged**